coq/coq-compile-common.el

```diff
@@ -20,6 +20,8 @@
   ;;(defvar coq-pre-v85 nil)
   (defvar coq-confirm-external-compilation nil); defpacustom
   (defvar coq-compile-parallel-in-background nil) ; defpacustom
+  (defvar compilation-error-regexp-alist-alist nil) ; from compile.el
+  (defvar compilation-error-regexp-alist nil) ; from compile.el
```

Comment:
Yup; I just meant that you can omit the `nil`.  But wouldn't `(require 'compile)` work as well?